yapapi/contrib/service/socket_proxy.py

```diff
@@ -199,14 +199,14 @@ def __repr__(self):
 
     @property
     def app_key(self):
-        """The application key used to authorize access to `yagna`'s REST API."""
+        """Return the application key used to authorize access to `yagna`'s REST API."""
```

Comment:
hmm, it's not a _function_ ... it's a property... I believe it should be a noun-like not an action-like...